app/archivesspace/archivesspace.controller.js

```diff
@@ -72,10 +92,29 @@
             level: function() {
               return '';
             },
+            start_date: function() {
+              return new Date;
+            },
+            end_date: function() {
+              return new Date;
+            },
+            date_expression: function() {
+              return '';
+            },
+            note: function() {
+              return '';
+            },
           },
         });
         form.result.then(function(result) {
           result.levelOfDescription = result.level;
+          result.notes = [{
+            type: 'odd',
+            content: result.note,
+          }];
+          delete result['note'];
```

Comment:
Looking at it, I feel like there isn't *too* much duplication here, though there is a little. The "edit" path is concerned with normalizing existing records (and recovering from a failure to POST changes) while the "create" path is constructing a new object. The main similarity is the normalization of the `levelOfDescription` and `notes` keys; should I pull that into a helper, maybe?